.github/workflows/ci.yml

```diff
@@ -41,17 +45,16 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm install
-      - run: npm install sequelize@${{ matrix.sequelize-version }}
-      - run: npm run build
+      - run: yarn install --frozen-lockfile --ignore-engines
+      - run: yarn add sequelize@${{ matrix.sequelize-version }} --ignore-engines
       - run: docker-compose up -d ${DIALECT}
       - run: docker run --link ${DIALECT}:db -e CHECK_PORT=${SEQ_PORT::-1} -e CHECK_HOST=db --net cli_default giorgos/takis
-      - run: npm run test
+      - run: yarn test
   test-sqlite:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [10, 12, 14, 16]
+        node-version: [10, 16]
         sequelize-version: [5, latest]
```

Comment:
Should we include 7 here?

Reply:
If we go ahead with splitting things up in v6 and v7 then no. Otherwise yes